### DISCUSSION TITLE
Change serialized member name with serialization macros

### DISCUSSION THREAD

**Discussion by sbehnke** [nlohmann/json#4466] [Q&A]
Is there a way to change the serialized variable name but still use the `NLOHMANN_DEFINE_TYPE_...` macros? I do realize I could write custom serialization code for the entire class, but I just have one member that uses a name that is a C++ keyword so I cannot use it as a variable name in my class.

**Comment by gregmarr** [comment]
No, there's no way to do this with the macros.
